apps/rush-lib/src/logic/PackageJsonUpdater.ts

```diff
@@ -124,13 +124,9 @@ export class PackageJsonUpdater {
       variant
     } = options;
 
-    const implicitlyPinned: Map<string, string> = InstallHelpers.collectImplicitlyPreferredVersions(
-      this._rushConfiguration,
-      {
-        variant
-      }
+    const implicitlyPinned: Map<string, string> = this._rushConfiguration.getImplicitlyPreferredVersions(
+      variant
     );
```

Comment:
This looks like a breaking API change.

Reply:
Only breaking in the sense that our `InstallHelpers` class doesn't have this method anymore. This class was used only by `PackageJsonUpdater` and the `InstallManager` for the package manager you're using, and this just shunts it into the RushConfiguration. If any 3rd parties too a dependency on `InstallHelpers`, access to `RushConfiguration` to call this method would have already been required. The options never supported anything other than `variant`